playground/mocks/data/idp/idx/identify-with-device-code.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
this seems same as `identify-with-password.json` thus no need to create another mock.

Reply:
fixed